components/cloud-foundry/src/view/util/cf-events-table/cf-events-table.html

```diff
@@ -0,0 +1,48 @@
+  <div ng-show="cfeCtrl.isLoading" class="cf-event-table-loading">
+    <bounce-spinner classes="bounce-spinner-sm"></bounce-spinner>
+  </div>
+  <table class="table cf-events-table"
+        ng-class="{'cf-event-table-loading': cfeCtrl.isLoading}"
+        st-table="cfeCtrl.events"
+        st-pipe="cfeCtrl.middleware">
+    <thead>
+      <tr>
+        <td class="text-center"
+        st-template="plugins/cloud-foundry/view/util/cf-events-table/cf-events-table-paginator.html"
+        st-pagination=""
+        st-displayed-pages="5"
+        ng-init="options=cfeCtrl"
+        st-items-by-page="cfeCtrl.itemsPerPage" colspan="4">
+        </td>
+      </tr>
+      <tr >
+        <th translate class="column">cf.events.labels.timestamp</th>
+        <th translate class="column">cf.events.labels.type</th>
+        <th translate class="column">cf.events.labels.actor-name</th>
```

Comment:
Would it be possible to put min-width 90 on the type column and 110 and actor-name? leads to less bunched up titles and content